### PR TITLE
Add Windows support

### PR DESCRIPTION
**Pull request by klaussnd**
Adjust zbar recipe to build on Windows/MSVC using the libiconv Conan recipe

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 from conans import ConanFile, CMake, tools
 
-
 class ZBarConan(ConanFile):
```
Comment (david-sinuela-pix4d) [inline conanfile.py:3]: It is missing the package_id method.

File: conanfile.py [hunk 2]
```diff
@@ -1,6 +1,5 @@
...
     name = 'zbar'
     lib_version = '0.23.0'
```
Comment (david-sinuela-pix4d) [inline conanfile.py:5]: Don't forget to bump the recipe revision

**Review by david-sinuela-pix4d — APPROVED** [review]
pre-approving but a couple of things are missing:

File: conanfile.py (hunk 2 above)
Comment (klaussnd) [reply conanfile.py:5]: The package is not yet used, so we don't need to bump the revision.

File: conanfile.py (hunk 1 above)
Comment (klaussnd) [reply conanfile.py:3]: fixed

**Review by imrodriguezro — APPROVED** [review]

**Pull request merged**